### PR TITLE
Add recipes for boto3 and AWS Greengrass Setup

### PR DESCRIPTION
**Pull request by akiernan**

### CODE REVIEW

File: recipes-aws/python/python-gg-group-setup.inc [hunk 1]
```diff
@@ -0,0 +1,27 @@
+SUMMARY = "A file-driven approach to the creation of an entire AWS Greengrass group"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=e3fc50a88d0a364313df4b21ef20c29e"
+
```
Comment (srware) [inline recipes-aws/python/python-gg-group-setup.inc]: Can you also add author and homepage for consistency.

File: recipes-aws/python/python-gg-group-setup.inc [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+PYPI_PACKAGE = "gg_group_setup"
+
+SRC_URI[md5sum] = "afc38bf424421634bea65fe32f2a32dd"
+SRC_URI[sha256sum] = "5d68dc5007a1bf24e8d1415db93e4f2b4dde42ffd98b868778bb3f6522d4d89d"
+
+DEPENDS += " \
+    ${PYTHON_PN}-boto3 \
+    ${PYTHON_PN}-fire \
+"
```
Comment (srware) [inline recipes-aws/python/python-gg-group-setup.inc]: Is this necessary for the package to build or are runtime dependencies enough?

File: recipes-devtools/python/python-fire.inc [hunk 3]
```diff
@@ -0,0 +1,8 @@
+SUMMARY = "A library for automatically generating command line interfaces"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=ab892b2b62f951d424bdb712a72584cc"
+
+SRC_URI[md5sum] = "9f3a91c285b50351d039b6f0bb91aac3"
+SRC_URI[sha256sum] = "6865fefc6981a713d2ce56a2a2c92c56c729269f74a6cddd6f4b94d16ae084c9"
+
+inherit pypi
```
Comment (srware) [inline recipes-devtools/python/python-fire.inc]: There are missing runtime dependencies if the setup.py is to be believed. This package should depend on 'six' and 'termcolor'.

File: recipes-devtools/python/python-fire.inc [hunk 4]
```diff
@@ -0,0 +1,8 @@
+SUMMARY = "A library for automatically generating command line interfaces"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=ab892b2b62f951d424bdb712a72584cc"
+
```
Comment (srware) [inline recipes-devtools/python/python-fire.inc]: Author and Homepage also please.

File: recipes-aws/python/python-gg-group-setup.inc (hunk 2 above)
Comment (akiernan) [reply recipes-aws/python/python-gg-group-setup.inc]: That'll teach me to push things upstream I did ages ago without checking properly! The RDEPENDS are fine.

**Pull request merged**